packages/samples/headless-ssr/src/components/hydration-metadata.tsx

```diff
@@ -0,0 +1,33 @@
+import {FunctionComponent} from 'react';
+import {SearchCSRState, SearchSSRState} from '../common/engine';
+
+export interface HydrationMetadataProps {
+  initialState: SearchSSRState;
```

Comment:
Could we rename `initial` -> `SSR` here and other occurrences in this PR for consistency?